src/Uno.UI.RemoteControl.VS/EntryPoint.cs

```diff
@@ -289,6 +313,21 @@ public void SetGlobalProperties(string projectFullName, IDictionary<string, stri
 		private void SetGlobalProperty(Microsoft.Build.Evaluation.Project msbuildProject, string propertyName, string propertyValue)
 		{
 			msbuildProject.SetGlobalProperty(propertyName, propertyValue);
+
+		}
+
+		private bool IsApplication(Microsoft.Build.Evaluation.Project project)
```

Comment:
This code snippet seems to be proliferating...

Reply:
It's not for the same types, but I agree. This will need to be adjusted somehow.